src/styles/_media.scss

```diff
@@ -0,0 +1,56 @@
+@use './variables.scss' as *;
+
+@media (max-width: 480px) {
+    .topHeader {
+        margin-bottom: 5%;
+    }
+
+    .bodyDiv {
+        margin-bottom: 6%;
+        min-height: 65%;
+    }
+
+    .introText {
+            margin-left: 4%;
+            margin-right: 4%;
+            line-height: 2.5rem;
+            font-size: 1.6rem;
+    }
+
+    .questionDiv {
+        width: 75%;
+        height: 60%;
+        padding-top: 5%;
+        padding-bottom: 5%;
+    }
+
+    .shipDiv {
+        bottom: 13%;
+    }
+
+    .image-container {
+        width: 110%;
+        top: -15%;
+        right: -120%;
+    }
+
+    .animate1 {
+        animation: scroll2 4s ease-in;
+        animation-iteration-count: 1;
+    }
+
+    @keyframes scroll2 {
+        from {
+            transform: translateX(0%)
+        }
+
+        to {
+            transform: translateX(-200%);
+            /* Move the background image to the left */
+        }
+    }
+
+    .answerButton {
+        margin-top: 8%;
+    }
```

Comment:
Great responsiveness right down to 320px - looks great and you did a great job avoiding elements getting cramped at smaller screen dimensions!